docs/proposals/multi-node/README.md

```diff
@@ -716,7 +775,7 @@ Mark the status condition type `AllMembersReady` to `true`.
 
 Mark the status condition type `AllMembersReady` to `false`.
 
-### 4. Majority members are `Ready` but `Ready` condition is stale
+### 5. Majority members are `Ready` but `Ready` condition is stale
```

Comment:
Can you elaborate how that can happen? The `Ready` condition is deduced right after the `members` are checked. Thus, the `Ready` condition should never be stale.

Reply:
In the proposal, I didn't make any assumptions about the order in which `members` and `conditions` are updated and left it for implementation. Hence, retained this section. But you are right, if `conditions` are computed after `members` are computed then this case would be redundant (also the the _Majority members are NotReady but Ready condition is stale_ case)?

Is it OK to retain these two cases just to allow for different implementations?

Reply:
Fair enough, let's keep it. Thanks for explaining.